src/Symfony/Bridge/Twig/Node/SearchAndRenderBlockNode.php

```diff
@@ -97,7 +98,12 @@ public function compile(Compiler $compiler): void
 
                         // Check at runtime whether the label is empty.
                         // If not, add it to the array at runtime.
-                        $compiler->raw('(twig_test_empty($_label_ = ');
+                        if (method_exists(CoreExtension::class, 'testEmpty')) {
+                            $compiler->raw('(Twig\Extension\CoreExtension::testEmpty($_label_ = ');
```

Comment:
No need to use the fully qualified class name here: https://github.com/twigphp/Twig/blob/4.x/src/Node/ModuleNode.php#L146